doc/api/http.md

```diff
@@ -3437,6 +3445,10 @@ changes:
   * `uniqueHeaders` {Array} A list of request headers that should be sent
     only once. If the header's value is an array, the items will be joined
     using `; `.
+  * `joinDuplicateHeaders` {boolean} It joins the field line values of
+    multiple headers in a request with ` ,` instead of discarding
```

Comment:
there should be a space after comma. This documentation is wrong.

Reply:
Yes this is wrong it should be `,  `